lib/fluent/root_agent.rb

```diff
@@ -55,9 +55,11 @@ def initialize(log:, system_config: SystemConfig.new)
       @suppress_emit_error_log_interval = 0
       @next_emit_error_log_time = nil
       @without_source = false
+      @enable_input_metrics = false
 
       suppress_interval(system_config.emit_error_log_interval) unless system_config.emit_error_log_interval.nil?
       @without_source = system_config.without_source unless system_config.without_source.nil?
+      @enable_input_metrics = system_config.enable_input_metrics unless system_config.enable_input_metrics.nil?
```

Comment:
How about `!!system_config.enable_input_metrics.nil?` to simplify it?

Reply:
I think the suggested changes does not have equal meaning:

```irb
irb(main):001:0> value = nil
=> nil
irb(main):002:0> !!value.nil?
=> true
irb(main):003:0> value = false
=> false
irb(main):004:0> !!value.nil?
=> false
irb(main):005:0> value = true
=> true
irb(main):006:0> !!value.nil?
=> false
```

This is not intended.

Reply:
Oops, sorry I intended `!!system_config.enable_input_metrics`.

Reply:
Hehe, no worries. I'll use it.

Reply:
Fixed in https://github.com/fluent/fluentd/pull/3411/commits/b00397af5962cf25bb3c3ba245d6799bae63b472.